src/js/coblocks-events-front.js

```diff
@@ -0,0 +1,46 @@
+( function( $ ) {
+	'use strict';
+
+	const moreEvents = $( '.wp-block-coblocks-events__more-events-wrapper' );
+	const nextPageIconContainer = $( '<div/>', { class: 'coblocks-events__next-page-icon-container' } );
+	const previousPageIconContainer = $( '<div/>', { class: 'coblocks-events__previous-page-icon-container' } );
+	const nextPageIcon = $( '<div/>', { class: 'next-page-icon' } );
+	const previousPageIcon = $( '<div/>', { class: 'previous-page-icon' } );
+	let currentPage = 0;
+
+	nextPageIconContainer.append( nextPageIcon );
+	previousPageIconContainer.append( previousPageIcon );
+
+	if ( moreEvents ) {
+		moreEvents.prepend( previousPageIconContainer );
+		moreEvents.append( nextPageIconContainer );
+	}
+
+	// Initially set only first page items to be shown
+	$( '.wp-block-coblocks-event-item' ).css( 'display', 'none' );
+	$( '.wp-block-coblocks-event-item[data-page="' + currentPage + '"]' ).css( 'display', 'initial' );
+
+	previousPageIconContainer.css( 'display', 'none' );
+
+	const totalPageCount = Number( $( '.wp-block-coblocks-event-item' ).last().attr( 'data-page' ) );
```

Comment:
We should count the elements with the class, not check a pre-counted value from an attribute on the last element. We can simply count the number of elements with `document.getElementsByClassName( 'wp-block-coblocks-event-item' ).length`.